pkg/operator/ceph/config/datapath.go

```diff
@@ -47,35 +53,30 @@ type DataPathMap struct {
 // which may include data from other daemons.
 func NewStatefulDaemonDataPathMap(
 	dataDirHostPath, daemonDataDirHostRelativePath string,
-	daemonType DaemonType, daemonID string,
+	daemonType DaemonType, daemonID, namespace string,
 ) *DataPathMap {
 	return &DataPathMap{
 		PersistData:      true,
 		HostDataDir:      path.Join(dataDirHostPath, daemonDataDirHostRelativePath),
 		NoData:           false,
 		ContainerDataDir: cephDataDir(daemonType, daemonID),
+		HostLogDir:       path.Join(dataDirHostPath, "log", namespace),
+		ContainerLogDir:  VarLogCephDir,
 	}
 }
 
 // NewStatelessDaemonDataPathMap returns a new DataPathMap for a daemon which does not persist data
 // to the host (mgrs, mdses, rgws)
 func NewStatelessDaemonDataPathMap(
-	daemonType DaemonType, daemonID string,
+	daemonType DaemonType, daemonID, namespace, dataDirHostPath string,
 ) *DataPathMap {
 	return &DataPathMap{
 		PersistData:      false,
 		HostDataDir:      "",
 		NoData:           false,
 		ContainerDataDir: cephDataDir(daemonType, daemonID),
-	}
-}
-
-func NewDatalessDaemonDataPathMap() *DataPathMap {
```

Comment:
It looks like I forgot to comment this when I created it with the rbd-mirror config refactor. It probably would have been a little clearer if I had done that. Perhaps you could add a comment to the effect of 
```go
NewDatalessDaemonDataPathMap returns a new DataPathMap for a daemon which does not utilize a data dir in the container as the mon, mgr, osd, mds, and rgw daemons do
```